packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -119,16 +120,6 @@ const ReactQuillEditor = ({
       ...newContent,
       ___isMarkdown: isMarkdownEnabled,
     } as SerializableDeltaStatic);
-
-    // sets the correct cursor position after pasting content
```

Comment:
Why the need to remove?

Reply:
@jnaviask removed because that was the code that was supposed to fix the copy/paste bug, but introduced the no-return cursor bug that I was working on.

Reply:
you fixed your bug but also we have a regression => https://github.com/hicommonwealth/commonwealth/pull/5783

https://github.com/hicommonwealth/commonwealth/assets/14819225/695fd050-1427-418b-9322-615ffed5a44d